python/pfs/drp/stella/reduceExposure.py

```diff
@@ -199,16 +199,18 @@ def run(self, sensorRefList):
 
         fiberTraceList = [sensorRef.get("fibertrace") for sensorRef in sensorRefList]
         detectorMapList = [sensorRef.get("detectormap") for sensorRef in sensorRefList]
-        pfsConfig = sensorRefList[0].get("pfsConfig")
+        pfsConfig = None
```

Comment:
Why is this part of PIPE2D-338?

Reply:
It deals with the original problem in a different way. Instead of constructing a `pfsConfig` for the data, this realises that we didn't need it in the first place (for some operations).